src/main/java/io/jenkins/plugins/appcenter/AppCenterRecorder.java

```diff
@@ -99,6 +102,10 @@ public String getReleaseNotes() {
         return Util.fixNull(releaseNotes);
     }
 
+    public String getNotifyTesters() {
+        return Boolean.toString(notifyTesters);
```

Comment:
Why does this need to be converted to a String only to be converted back again to a boolean later?

Reply:
The `envVars.expand()` in `UploadRequest` only takes a string. Can you recommend a better way? Still getting to know the code here and my Java is rusty 😝

Reply:
Ahh, OK. I see. In that case I feel that not everything needs to have an environment variable expansion. So keep it as a boolean  defaulted to true and pass that straight on.

Reply:
Whilst the other fields are more naturally represented as strings and it makes sense to offer variable expansion (Jenkins users love variable expansions) for this field I don't think it does. I've changed this to simply pass through the configured value which will default to `true` if not explicitly set to `false.